arcwelder_settings.def.json

```diff
@@ -147,10 +147,10 @@
             },
             "arcwelder_min_arc_segment":
             {
-                "label": "Minimum Arc Segments",
```

Comment:
I'm a bit torn on this. On the one hand, this change makes sense as it fits the description better. On the other hand, the current name is copied from the original Arc Welder lib source: 
https://github.com/FormerLurker/ArcWelderLib?tab=readme-ov-file#minimum-arc-segments

So should this plugin follow the original implementation (and as a result probably other tools and utilities that use ArcWelder) or should it try to do better (but risk confusing people).

Reply:
I think that it should align with whatever the user might have to be concerned with - and I think they would be more likely to be concerned (in this section) with their firmware settings than with some underlying library that they may not need to know the details of.

But this is a personal and subjective view and alternative viewpoints might have equally valid rationales.